### PR TITLE
Fix required field for SLO widget

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

* Fix optional field to required field

### CODE REVIEW

**Comment by KeisukeYamashita** [comment]
@cdebourcy  Could you take a look and merge, please? After this, we can push and merge the terraform-providers/terraform-provider-datadog:)

Thank you.

**Review by cdebourcy — APPROVED** [review]
What is this for?

File: website/docs/r/dashboard.html.markdown [hunk 1]
```diff
@@ -731,11 +731,11 @@ Nested `widget` blocks have the following structure:
         - `title_align`: (Optional) The alignment of the widget's title. One of "left", "center", or "right".
         - `time`: (Optional) Nested block describing the timeframe to use when displaying the widget. The structure of this block is described [below](dashboard.html#nested-widget-time-blocks).
   - `service_level_objective_definition`: The definition for a Service Level Objective widget. Exactly one nested block is allowed with the following structure:
-        - `view_type`: (Optional) Type of view to use when displaying the widget. Only "detail" is currently supported.
```
Comment (kuannie1) [inline website/docs/r/dashboard.html.markdown]: what is this for?

**Review by cdebourcy — CHANGES_REQUESTED** [review]
What is this for?

**Comment by mbarrien** [comment]
LGTM.

**Pull request merged**